README.md

```diff
@@ -13,86 +13,38 @@ TorchServe is a flexible and easy to use tool for serving PyTorch models.
 
 ## Install TorchServe
 
-Conda instructions are provided in more detail, but you may also use `pip` and `virtualenv` if that is your preference.
-**Note:** Java 11 is required. Instructions for installing Java 11 for Ubuntu or macOS are provided in the [Install with Conda](#install-with-conda) section.
-
-### Install with pip
-
-1. Install Java 11
-
-    ```bash
-    sudo apt-get install openjdk-11-jdk
-    ```
-
-1. Use `pip` to install TorchServe and the model archiver:
-
-    ``` bash
-    pip install torch torchtext torchvision sentencepiece psutil future
-    pip install torchserve torch-model-archiver
-    ```
-
-### Install with Conda
-**Note:** For Conda, Python 3.8 is required to run Torchserve
-
-#### Ubuntu
-
 1. Install Java 11
 
+    For Ubuntu
     ```bash
     sudo apt-get install openjdk-11-jdk
     ```
-
-1. Install Conda (https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html)
```

Comment:
We should not remove the Conda install instructions for Linux and Mac

Reply:
@maaquib : We have tried to simplify the installation steps as much as possible.

 - install java
   - This step is independent of python environment
 - install python dependencies through requirements.txt
   - This will always install the dependencies using pip irrespective of the conda or pip environment.
 - install `torchserve` and `torch-model-archiver` from `conda/PyPi` repo based on your environment.
   - This is the only step which requires conda/pip specific commands which we have added.

Guide to install conda only provides steps for setting up a python virtual environment which should not be a part of `torchserve` installation doc. Or Else we should have a separate section for setting up a virtual env using conda/pyenv/virtualenv/venv etc.

Thoughts?

Reply:
@harshbafna Just added review comment. That should be good enough. Let me know what you think.

Reply:
@maaquib : merged the review comments, resolving this conversation.